packages/react-aria-components/stories/index.stories.tsx

```diff
@@ -1246,3 +1246,76 @@ export const LinkExample = () => {
     </Link>
   );
 };
+
+export const ToolbarExample = (props) => {
+  return (
+    <Toolbar {...props}>
+      <ToggleButton className={classNames(styles, 'toggleButtonExample')}><strong>B</strong></ToggleButton>
+      <ToggleButton className={classNames(styles, 'toggleButtonExample')}><div style={{textDecoration: 'underline'}}>U</div></ToggleButton>
+      <ToggleButton className={classNames(styles, 'toggleButtonExample')}><i>I</i></ToggleButton>
+      {/* radio does not work in toolbar yet, selection doesn't follow focus
+      <RadioGroup orientation="horizontal" className={styles.radiogroup} aria-label="Text Alignment">
+        <Radio className={styles.radio} value="left">Left</Radio>
+        <Radio className={styles.radio} value="center">Center</Radio>
+        <Radio className={styles.radio} value="right">Right</Radio>
+      </RadioGroup>
+      */}
+      {/* disabled buttons do not work in toolbar yet, the buttons cannot be focused
+      <Toolbar>
+        <Button isDisabled>Copy</Button>
+        <Button isDisabled>Paste</Button>
+        <Button isDisabled>Cut</Button>
+      </Toolbar>
+      */}
+      {/* select does not work in toolbar yet, focus can't move "Right Arrow" past it
```

Comment:
if our listeners are capturing seems like it should?

Reply:
Determined what happened, I wasn't specifying to the focusmanager to move to the next tabbable element, so focus was going to the hidden select and then right back to the select button, forever